controllers/articles.js

```diff
@@ -364,12 +450,13 @@ const articles = {
       if (checkExist) {
         return res.status(400).send({
           status: res.statusCode,
-          errorMessage: 'You have already bookmarked this article',
+          errorMessage: 'You have already bookmarked this article'
```

Comment:
@avpaul I have noticed, you have kept on removing the commas at the end of each comment, I think it's healthy to keep them there in order to avoid an error when refactoring

Reply:
@avpaul Thanks for the initiative of working on the mock-up for this sprint.

Reply:
Actually for the rules we are following, we don't have to put the comma on the last object element. check our .eslintrc file